docs/portal/image-registry/managing-repositories-under-a-project.md

```diff
@@ -9,4 +9,4 @@ The following procedure describes how to view repositories under a project, how
 <p align=center><img src="/docs/resources/images/registry/edit-repository.png" width="600"></p>
```

Comment:
The 2 sections above do not have same indentaion and bullet point style. Please check.

Reply:
Fixed.